service-tests/github.js

```diff
@@ -132,6 +132,20 @@ t.create('Tag')
     value: Joi.string()
   }));
 
+t.create('Package version')
+  .get('/package/badges/shields.json')
+  .expectJSONTypes(Joi.object().keys({
+    name: Joi.equal('development'),
+    value: Joi.string().regex(/^v\d+(\.\d+)?(\.\d+)?$/)
+  }));
+
+t.create('Manifest version')
+  .get('/manifest/RedSparr0w/IndieGala-Helper.json')
+  .expectJSONTypes(Joi.object().keys({
+    name: Joi.equal('development'),
+    value: Joi.string().regex(/^v\d+(\.\d+)?(\.\d+)?$/)
+  }));
```

Comment:
Thanks for these tests! Great code.

How about changing the URL paths to `/package-json/` and `/manifest-json/`? To someone looking at the URL, that makes it quite a bit clearer what these are doing.

Reply:
Also, after reading #308, I think we should call these `/package-json/v/` and `/manifest-json/v/` so we can expand to pull other values out of those files, such as the node version.

Reply:
I don't know about the `/package-json/` part, not sure that would add much value due to it always being a json file?

Definitely agree on the `/package/v/` though this makes a lot of sense rather than just limiting to the version will allow more options.

Reply:
> I don't know about the `/package-json/` part, Not sure that would add much value due to it always being a json file?

What I mean is this: if I saw a readme that linked to `img.shields.io/package/something` I don't think I'd ever guess the badge was pulling from `package.json`.